.golangci.yaml

```diff
@@ -0,0 +1,30 @@
+run:
+  timeout: 5m
+  skip-dirs:
+  - pkg/lib
+  - pkg/api
+  - pkg/fakes
+  - pkg/package-server/apis
+  - test/e2e
```

Comment:
Removing this can be done in a follow-up PR.

Reply:
Can you explain the exclusion for these paths? I can see why we don't care about fakes and test but the rest are good to have.

Reply:
Yeah, definitely - I had a couple of reasons:

- Aim to reduce the reviewer burden with an already large changeset;
- Get the main infrastructure in place so we golangci-lint can identify linting issues ahead of time to reduce reviewer responsibilities going forward vs. fixing all lints all at once;
- I was already fairly burnt out identifying and manually fixing all the linting violations and didn't want to hold this up further, as I'm less concerned about test code